test/comment.test.js

```diff
@@ -0,0 +1,113 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import models from '../models';
+import server from '../app';
+
+chai.use(chaiHttp);
+
+describe('COMMENT TEST SUITE', () => {
+  let accesstoken;
+  let slug;
+  const comment = 'This is in insightful article';
+  before(async () => {
+    await models.sequelize.sync({ force: true });
+
+    const userRequestObject = {
+      username: 'newusername',
+      email: 'newaddress@email.com',
+      password: 'newpassword'
+    };
+    const articleRequestObject = {
+      title: 'Post to test if article already exists',
+      body: 'Article Body',
+      description: 'Article Description',
+      category: 'Science'
+    };
+    slug = 'post-to-test-if-article-already-exists';
+    const responseObject = await chai.request(server).post('/api/v1/users')
+      .send(userRequestObject);
+    accesstoken = responseObject.body.token;
+
+    await chai.request(server)
```

Comment:
You should get the slug from the result of this. You shouldn't need to assign it to an hardcoded string as you did on line 26.

Reply:
Just realized you need more to make this work. Please add `slug` to the response of the `createArticle` controller so that you can access it here in the tests.

Reply:
Alright